00-preface.Rmd

```diff
@@ -11,7 +11,7 @@
 マクロ数量分析の標準的なツールである Dynare を自分で使ったり, 名前を聞いたことのある学生も多いと思う. 一方で, その裏側でどのような計算が行われているかを知らないという学生も多いのではないかと思う. 理論に無関心でもおおよそ問題が起こらないのは, ひとえに Dynare が優れたインターフェイスを備えたアプリケーションであるということに外ならないのだが, そのブラックボックスを開けて理論を理解しようというのがこの講義の第1の課題である. 
 
 第2の課題は, 読者にコンピュータシミュレーションの基礎的な手法を身につけてもらうことである. 線形合理的期待モデルは近年のマクロ経済の数量分析に大きな役割を果たしているから, 各自の研究プロジェクトに大いに役に立つものと信じている. 
-数学もプログラミングも自分の手を動かし, 頭を悩ませなければ習熟できない. 理論とその実装を行き来しながら総合的な問題解決能力を養ってほしい. この講義では, 主に R言語を用いる.^[R ([https://www.r-project.org/](https://www.r-project.org/)) と RStudio ([https://www.rstudio.com/](https://www.rstudio.com/)) の最新版を各自インストールしてほしい. R言語を選んだ理由は, 計量経済学の学習のために利用したことのある学生が多いだろう考えたからであり, R がシミュレーションのために最適であるとは考えていない. 大変人気のあるプログラミング言語・環境であるから一度は学んでおいて損はないと思う.]
+数学もプログラミングも自分の手を動かし, 頭を悩ませなければ習熟できない. 理論とその実装を行き来しながら総合的な問題解決能力を養ってほしい. この講義では, 主に R言語を用いる.^[[R](https://www.r-project.org/) と [RStudio](https://www.rstudio.com/) の最新版を各自インストールしてほしい. R言語を選んだ理由は, 計量経済学の学習のために利用したことのある学生が多いだろう考えたからであり, R がシミュレーションのために最適であるとは考えていない. 大変人気のあるプログラミング言語・環境であるから一度は学んでおいて損はないと思う.]
```

Comment:
I understand what you mean. I'm trying to make this work look nice in print. That's why I wanted to show the URLs.